nilearn/decomposition/canica.py

```diff
@@ -146,20 +147,28 @@ def fit(self, imgs, y=None, confounds=None):
             This parameter is passed to nilearn.signal.clean. Please see the
             related documentation for details
         """
+        # Fit MultiPCA without sorting
+        sorted = self.sorted
+        self.sorted = False
```

Comment:
Yes, this is only to avoid sorting in MultiPCA, which is useless as we will sort components at the end of ICA